helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -404,6 +404,14 @@ starrocksBeSpec:
   # If runAsNonRoot is true, the container is run as non-root user.
   # The userId will be set to 1000, and the groupID will be set to 1000.
   runAsNonRoot: false
+  # add/drop capabilities for BE container, DataDog profiling lib relies on it for example. It could be added as below:
+  # capabilities:
+  #    add:
+  #      - PERFMON
+  #      - SYS_PTRACE
+  #    drop:
+  #      - SYS_ADMIN
+  capabilities: {}
```

Comment:
Can you please add the example comment below `capabilities ` like other fileds ?